odoo/addons/base/models/res_users.py

```diff
@@ -199,6 +199,80 @@ def _search(self, args, offset=0, limit=None, order=None, count=False, access_ri
             return len(groups) if count else groups.ids
         return super(Groups, self)._search(args, offset=offset, limit=limit, order=order, count=count, access_rights_uid=access_rights_uid)
 
+    def _get_recursive_implied_groups(self, recursive_groups):
+        for group in self.implied_ids.filtered(
```

Comment:
is it technically possible to have loop in group ? :thinking: 
if yes, we might ensure that this method won't be called infinitely 

maybe with something like this ?
```python
if group in recursive_groups:
    continue
```